### PR TITLE
Added development guideline.

### PR DESCRIPTION
**Pull request by gokhangulbiz**

### CODE REVIEW

**Comment by gokhangulbiz** [comment]
Could you please take a look @daronyondem, @ilkayilknur?

**Comment by daronyondem** [comment]
I have done a couple of tiny changes and sent as a separated PR. https://github.com/XomniCloud/xomni-cms-vnext/pull/24

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  * Requires a MSSQL Database to provide navigation tree. Database generation and sample data script can be found in **scripts** folder.
+  * Requires accessing XOMNI Master APIs to provide license details. You need to enter **masterAPIUserName**, **masterAPIPassword**, **masterAPIUri** key values defined in web.config
+  * Can be simply build with Visual Studio 2013.
+
+##XOMNI.CMS.Backend.Test
+  * Contains Backend project unit tests.
+
+##XOMNI.CMS.FrontEnd
+  * Contains CMS vNext UI.
+  * Dependent Backend project to show license cap and navigation data. (Backend API url configuration can be foun in **src/appSettings.json**)
```
Comment (ilkayilknur) [inline DEVELOPMENT.md]: foun=> found typo fix.

**Comment by ilkayilknur** [comment]
Should we write something about config based authentication that we've implemented this sprint ? Otherwise, contributers may think that this project dependent on our old CMS. What do you think ?

**Comment by gokhangulbiz** [comment]
Thanks @daronyondem,  @ilkayilknur! I have merged #24 into #23. And added a new commit about how to by-pass cookie based authentication by changing appsettings.

**Comment by ilkayilknur** [comment]
@gokhangulbiz If @daronyondem won't make any changes for your last commit, I think we can close this PR and get this document into the repository.

**Comment by gokhangulbiz** [comment]
Merged.

**Pull request closed without merging**